pkg/commands/workload_test.go

```diff
@@ -1610,6 +1611,16 @@ func TestWorkloadOptionsPublishLocalSource(t *testing.T) {
 		shouldError    bool
 		expectedOutput string
 	}{{
+		name:     "local source with excluded files",
```

Comment:
can you add a test or update this test with tanzuignore file 
- that includes a dir path that exists
- a nested file in the dir thats excluded

Reply:
Test updated